README.md

```diff
@@ -1 +1,179 @@
-# LearningJournal
\ No newline at end of file
+# LearningJournal  
+
+## My WebSite URL  
+- https://regenal-learning-journal.herokuapp.com/
+
+## Getting Started
+
+- git clone repo
+- $VENV/bin/activate
+- cd LearningJournal/learning_journal_basic
+- pip install -e .
+- pserve development.ini
+
+## Navigating
+
+- path to get to home page --> /
+- path to get to detail page --> /journal/{id:\d+}
+- path to get to create page --> /journal/new-entry
+- path to get to update page --> /journal/{id:\d+}/edit-entry
+
+## Links
+
+- Home button is located on each page, returns to lists view.
+- New-entry button is located on each page, returns to create view.
+- title are links that take you to the detail page
+
+
+## Coverage Reports  
+
+### coverage report step1
+
+---------- coverage: platform darwin, python 2.7.10-final-0 ----------
```

Comment:
I don't think these are being rendered in your README on GitHub like you wanted them to be... try indenting the entire block of test results twice and view that.  If you have Atom installed, you can use the markdown preview to see your md files, updated in as you save changes.